playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -46,12 +46,16 @@
 
   # Replace dc/docker-registry certificate secret contents if set.
   - block:
+    - name: Load lib_openshift modules
```

Comment:
What is the right place for the include_role is something we have not figured out yet. I prefer the explicit include than failing when the role is missing.